api/envoy/extensions/filters/network/thrift_proxy/router/v3/router.proto

```diff
@@ -18,4 +20,7 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 message Router {
   option (udpa.annotations.versioning).previous_message_type =
       "envoy.config.filter.thrift.router.v2alpha1.Router";
+
+  // Close downstream connection in case of routing or upstream connection problem. Default: true
```

Comment:
Should probably leave a pointer to the runtime flag here.

Reply:
I think there is a misunderstanding here. The default is `true` (close the downstream) which is the original behavior. So if this field is not set then there is no change in the working either.
Maybe it is a little unusual to use true as default for such flag, but I made like this according to comment:
https://github.com/envoyproxy/envoy/pull/20756#discussion_r847439358

Reply:
Yeah this is on by default (current behavior) so I think we should be good. This is less controversial than #20236, so current behavior by default should suffice and save us from needing a runtime flag.

Reply:
I see. Sounds good.